### PR TITLE
Fix warnings in the sample app

### PR DESCRIPTION
**Pull request by boxtcarpel**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @boxtcarpel has signed the CLA. Thanks for the pull request!

File: BoxContentSDKSampleApp/BoxContentSDKSampleApp/Base.lproj/LaunchScreen.xib [hunk 1]
```diff
@@ -11,7 +15,7 @@
             <rect key="frame" x="0.0" y="0.0" width="480" height="480"/>
             <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
             <subviews>
-                <label opaque="NO" clipsSubviews="YES" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="  Copyright (c) 2015 Box. All rights reserved." textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" minimumFontSize="9" translatesAutoresizingMaskIntoConstraints="NO" id="8ie-xW-0ye">
+                <label opaque="NO" clipsSubviews="YES" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="  Copyright (c) 2017 Box. All rights reserved." textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" minimumFontSize="9" translatesAutoresizingMaskIntoConstraints="NO" id="8ie-xW-0ye">
```
Comment (boxtcarpel) [inline BoxContentSDKSampleApp/BoxContentSDKSampleApp/Base.lproj/LaunchScreen.xib:18]: Updated 2015 to 2017

File: BoxContentSDKSampleApp/BoxContentSDKSampleApp/Info.plist [hunk 2]
```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSPhotoLibraryUsageDescription</key>
```
Comment (boxtcarpel) [inline BoxContentSDKSampleApp/BoxContentSDKSampleApp/Info.plist:5]: Prevent crash in the photo import upload of the Sample app

File: BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAccountsViewController.m [hunk 3]
```diff
@@ -65,12 +65,19 @@ - (void)barButtonPressed:(UIBarButtonItem *)sender
             if ([error.domain isEqualToString:BOXContentSDKErrorDomain] && error.code == BOXContentSDKAPIUserCancelledError) {
                 BOXLog(@"Authentication was cancelled, please try again.");
             } else {
-                UIAlertView *alertView = [[UIAlertView alloc] initWithTitle:nil
-                                                                    message:@"Login failed, please try again"
-                                                                   delegate:nil
-                                                          cancelButtonTitle:nil
-                                                          otherButtonTitles:@"OK", nil];
-                [alertView show];
+                UIAlertController *alertController = [UIAlertController alertControllerWithTitle:nil
```
Comment (boxtcarpel) [inline BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAccountsViewController.m:68]: Most of the fixes are in moving to a UIAlertController (from UIAlertView)

**Comment by thuyBox** [comment]
+2

**Pull request merged**